internal/kafka/command_broker.go

```diff
@@ -14,7 +14,7 @@ func newBrokerCommand(prerunner pcmd.PreRunner) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:         "broker",
 		Short:       "Manage Kafka brokers.",
-		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireOnPremLogin},
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonCloudLogin},
```

Comment:
This naming seems misleading. We only require login for Cloud, but not for On-Prem. Should we have an annotation here at all?

Reply:
The name is definitely misleading. I checked the logic of that annotation, and what it's doing is return an error if the user is logged into Cloud.

It doesn't error if the user is logged out or logged into MDS.

I believe that's what we want here: the broker commands are on-prem only so we want to block Cloud users here

Reply:
Yeah exactly, so instead of `RequireNonCloudLogin` we want `RequireNotCloudLogin` lol